go.mod

```diff
@@ -1,17 +1,10 @@
-module github.com/openlyinc/openlynt
+module go.openly.dev/openlynt
 
 go 1.13
```

Comment:
this is soooo old, i'd bump it to at least 1.19. modules work differently with old versions.